sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/prometheus/ProducerTopicMessageCounter.java

```diff
@@ -11,7 +11,7 @@
 public class ProducerTopicMessageCounter {
 
   /** The counter name as it is published to Prometheus. */
-  private static final String METRIC_NAME = "kafka_producer_topic_message_total";
+  private static final String METRIC_NAME = "kafka_producer_topic_message";
```

Comment:
If I see that correctly, the metric containing the total count value will now be `kafka_producer_topic_message_count`.
@SDA-SE/sda-commons-maintainer is that a breaking change or is it enough to mention that as important in the release notes?

Reply:
My fault, total is appended automatically now and I was confused by the _duration_count